docker-compose.yml

```diff
@@ -17,24 +26,23 @@ services:
       interval: 5s
       timeout: 60s
       retries: 5
-  emx2:
+  molgenis-emx2:
     image: molgenis/molgenis-emx2
-    environment:
-    - MOLGENIS_POSTGRES_URI=jdbc:postgresql://postgres/molgenis
-    - MOLGENIS_POSTGRES_USER=molgenis
-    - MOLGENIS_POSTGRES_PASS=molgenis
-    #      - MOLGENIS_OIDC_CLIENT_ID=[]
-    #      - MOLGENIS_OIDC_CLIENT_SECRET=[]
-    #      - MOLGENIS_OIDC_CLIENT_NAME=MolgenisAuth
-    #      - MOLGENIS_OIDC_DISCOVERY_URI=https://auth.molgenis.org/.well-known/openid-configuration/
-    #      - MOLGENIS_OIDC_CALLBACK_URL=http://localhost:8080
```

Comment:
personally i prevere the inline env setting over the env file , less magic, the one file thing is a strong point of the dockercompose system

Reply:
I agree. Most users will find that easier too.